cub/cub/detail/launcher/cuda_driver.cuh

```diff
@@ -27,6 +27,7 @@ struct CudaDriverLauncher
   dim3 block;
   std::size_t shared_mem;
   CUstream stream;
+  bool dependent_launch; // TODO: this is not being used currently
```

Comment:
I think you need to change the call to `cuLaunchKernel` below to something like:
```c++
if (dependent_launch) {
  cudaLaunchAttribute attribute[1];
  attribute[0].id                                         = cudaLaunchAttributeProgrammaticStreamSerialization;
  attribute[0].val.programmaticStreamSerializationAllowed = 1;

  cudaLaunchConfig_t config{};
  config.gridDim          = grid;
  config.blockDim         = block;
  config.dynamicSmemBytes = shared_mem;
  config.stream           = stream;
  config.attrs            = attribute;
  config.numAttrs         = 1;
  cudaLaunchKernelEx(&config, k, args...);
} else {
  // current code path
}
```

Reply:
I added this, but I also added similar checks to the ones here https://github.com/NVIDIA/cccl/blob/34af9a490eeadf2b392d62702bfb543dcbe46712/thrust/thrust/system/cuda/detail/core/triple_chevron_launch.h#L89

Reply:
Great!